thinc/layers/remap_ids.py

```diff
@@ -31,11 +41,23 @@ def forward(
     table = model.attrs["mapping_table"]
     default = model.attrs["default"]
     dtype = model.attrs["dtype"]
+    column = model.attrs["column"]
+    if column is not None:
+        inputs = cast(Array2d, inputs)
+        inputs = inputs[:, column]
+    # elements of cupy arrays are 0-dimensional arrays
+    # not the integers stored in the original mapper.
+    if is_cupy_array(inputs):  # type: ignore
+        if inputs.dtype.kind == 'i':  # type: ignore
+            inputs = map(int, inputs)  # type: ignore
```

Comment:
Doesn't this retrieve the elements from the GPU one by one? Convert everything all at once as a NumPy array.